internal/common/foraging/setup.go

```diff
@@ -7,12 +7,12 @@ import (
 )
 
 // CreateDeerHunt receives hunt participants and their contributions and returns a DeerHunt
-func CreateDeerHunt(teamResourceInputs map[shared.ClientID]shared.Resources, dhConf config.DeerHuntConfig) (DeerHunt, error) {
+func CreateDeerHunt(teamResourceInputs map[shared.ClientID]shared.Resources, dhConf config.DeerHuntConfig, logger shared.Logger) (DeerHunt, error) {
 	if len(teamResourceInputs) == 0 {
 		return DeerHunt{}, errors.Errorf("No deer hunt resource contributions specified!")
 	}
 	params := deerHuntParams{p: dhConf.BernoulliProb, lam: dhConf.ExponentialRate}
-	return DeerHunt{ParticipantContributions: teamResourceInputs, params: params}, nil // returning error too for future use
+	return DeerHunt{ParticipantContributions: teamResourceInputs, params: params, logger: logger}, nil // returning error too for future use
```

Comment:
No problem but note that this causes a runtime error if you try to log within the test. Usually we make a mock logger to save logs in a string so that we can _test logs_--a neat test method

Reply:
Don't need to change though, it's just a test

Reply:
Cool - thanks for the heads up